src/components/config.js

```diff
@@ -0,0 +1,10 @@
+let loadedConfig
+
+const loadConfig = () =>
+  fetch('config.json').then(response => response.json()).then(json => loadedConfig = json)
+
+
+const getRawlsUrlRoot = () => loadedConfig['rawlsUrlRoot']
+const getGoogleClientId = () => loadedConfig['googleClientId']
```

Comment:
Same suggestion about alphabetizing.

Reply:
This is the order they appear in inside of the config.json itself.

Reply:
Okay, let's merge this and reorder those in a follow-up.